src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -67,16 +68,28 @@ private void UpdateInstalledVersions()
 
             foreach (var project in _projects)
             {
-                var installedVersion = GetInstalledPackage(project.NuGetProject, Id);
-                if (installedVersion != null)
+                try
                 {
-                    project.InstalledVersion = installedVersion.PackageIdentity.Version;
-                    hash.Add(installedVersion.PackageIdentity.Version);
-                    project.AutoReferenced = (installedVersion as BuildIntegratedPackageReference)?.Dependency?.AutoReferenced == true;
+                    var installedVersion = GetInstalledPackage(project.NuGetProject, Id);
+                    if (installedVersion != null)
+                    {
+                        project.InstalledVersion = installedVersion.PackageIdentity.Version;
+                        hash.Add(installedVersion.PackageIdentity.Version);
+                        project.AutoReferenced = (installedVersion as BuildIntegratedPackageReference)?.Dependency?.AutoReferenced == true;
+                    }
+                    else
+                    {
+                        project.InstalledVersion = null;
+                    }
                 }
-                else
+                catch(Exception ex)
```

Comment:
Not really, because we want to avoid any exception thrown from `GetInstalledPackage` at this time. We already takes care of it during PackagesErrorBar and shows appropriate error message there.